src/features/theme/ui/GiftThemes.tsx

```diff
@@ -0,0 +1,36 @@
+import { GiftCategoryGrid, GiftCategoryItem } from "@/entities/gift/ui";
+
+import { useGiftThemes } from "@/features/theme/services/getGiftThemes";
+
+import { Spinner } from "@/shared/ui/Spinner";
+
+export const GiftThemes = () => {
+    const { isPending, data, error } = useGiftThemes();
+
+    if (isPending) {
+        return (
+            <GiftCategoryGrid>
+                <Spinner size="40px" borderWidth="4px" color="#000" />
+            </GiftCategoryGrid>
+        );
+    }
+
+    if (error) {
+        return <GiftCategoryGrid>에러 발생</GiftCategoryGrid>;
+    }
+
+    return (
+        <GiftCategoryGrid>
+            {data &&
+                data.map((category) => {
```

Comment:
optional chaining을 사용해보셔도 좋을거 같아요!
```suggestion
            { data?.map((category) => {
```